airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -100,6 +100,8 @@
 
 public abstract class DestinationAcceptanceTest {
 
+  private static final ArrayList<String> TEST_SCHEMAS = new ArrayList<>();
```

Comment:
```suggestion
  private static final Set<String> TEST_SCHEMAS = new HashSet<>();
```

then you don't need the `TEST_SCHEMAS.contains` check